app/assets/stylesheets/workspace.scss

```diff
@@ -1,6 +1,14 @@
+.workspaces-show {
+  .container-fluid {
+    margin: 0;
+    padding: 0;
+  }
+}
+
 #viewer {
+  clear: both;
   display: block;
-  height: 600px;
-  position: relative;
+  height: calc(100% - 55px); // should be $navbar-height, but variable is not available
```

Comment:
I think this may be different in Bootstrap 4 which is included. Perhaps https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss#L600 ?